### DISCUSSION TITLE
fritzbox 6490 fritz.os 7.20

### DISCUSSION THREAD

**Discussion by citarone** [Freetz-NG/freetz-ng#246] [Allgemein]
hi I have the problem that the freetz firmware cannot be installed. get the message that it is not an original firmware. tried to downgrade got the same message what else can i try.

ps: am a noop please be as detailed as possible
pps: the fritzbox is an old one that otherwise no longer has any function

**Comment by fda77** [comment]
https://freetz-ng.github.io/ >> README

**Comment by pony1k** [comment]
there is a tool for that.
`tools/push_firmware -h`
